.github/stale.yaml

```diff
@@ -23,8 +23,8 @@ limitPerRun: 5
 
 # Specify configuration settings that are specific to issues
 issues:
-  daysUntilStale: 60
-  daysUntilClose: 7
+  daysUntilStale: 7
```

Comment:
it would be better to keep it in reverse order, marking it as stale in only 7 days doesn't seem correct. we can mark it as stale for 25/30 days and close it after 7 days if there is no activity

Reply:
@Madhu-1 exactly that was the condition or previous to this PR. With this change the aim is to see the stale bot come in action after 7 days or not. If it happens, we could exchange the params. Otherwise we have to wait for another 25/30 days to see or check stale bot behaviiour. I dont see a mechanism to run a single trigger forcefully.

Reply:
I do not see the need for testing this. Can you not place an event in your calendar to check on an issue that is meant to become stale in 60 days?